### DISCUSSION TITLE
How do I translate this to English? 如何将这个翻译成英语？

### DISCUSSION THREAD

**Discussion by cometbus** [binary-husky/gpt_academic#334] [Q&A]
How do I translate this to English? 如何将这个翻译成英语？

**Comment by binary-husky** [comment]
@cometbus 
I uploaded a old version of translation here.

And you can use this plugin `https://github.com/binary-husky/chatgpt_academic/blob/master/crazy_functions/%E4%BB%A3%E7%A0%81%E9%87%8D%E5%86%99%E4%B8%BA%E5%85%A8%E8%8B%B1%E6%96%87_%E5%A4%9A%E7%BA%BF%E7%A8%8B.py` to do a full-project translation again on new version projects.

However, note that sometimes ChatGPT breaks code indent, you may still need to fix the indent here and there.

https://github.com/binary-husky/chatgpt_academic/issues/300
